specs/Separate_conf_files.adoc

```diff
@@ -0,0 +1,141 @@
+= Separate out per-component configuration files
+
+Configuration details for the tendrl modules are read from same file(tendrl.conf).
+Based on the tendrl modules we have to keep the configuration details into different
+files with proper name and read it independently.
+
+== Problem description
+
+Currently all the configuration details for different modules are read from same file .
```

Comment:
Also while import flow the configuration file gets appended with SDS specific configurations while installation of SDS integration bits on the storage nodes. This also should be mentioned in problem statement.